### PR TITLE
Support for Java fields

### PR DESCRIPTION
**Pull request by magidc**
I have extended Treessiter queries for java to capture Java fields. I have also extended Java test case file

### CODE REVIEW

File: queries/java/aerial.scm [hunk 1]
```diff
@@ -9,3 +9,9 @@
 
 (enum_declaration
   name: (identifier) @name) @type
+
+(field_declaration
+  type: [(integral_type) (type_identifier) (boolean_type)] @java_type
```
Comment (stevearc) [inline queries/java/aerial.scm:14]: nit: remove the `@java_type` capture group; I don't think it's used

File: tests/treesitter/java_spec.lua [hunk 2]
```diff
@@ -41,15 +41,33 @@ describe("treesitter java", function()
...
+            end_col = 22,
+          },
+          {
+            kind = "Field",
+            name = "field_2",
+            level = 1,
+            lnum = 8,
+            col = 2,
+            end_lnum = 8,
+            end_col = 21,
```
Comment (stevearc) [inline tests/treesitter/java_spec.lua:60]: Looks like tests expect this to be 24

**Review by stevearc — CHANGES_REQUESTED** [review]
Awesome, thanks for the PR! Couple small nits and it'll be good to merge

File: tests/treesitter/java_spec.lua (hunk 2 above)
Comment (magidc) [reply tests/treesitter/java_spec.lua:60]: yep, it should be 24, my fault

**Comment by stevearc** [comment]
Thanks again!

**Comment by magidc** [comment]
Thank you for developing Aerial! That was my first time dealing with TS queries

**Pull request merged**